edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/CustomsReference.java

```diff
@@ -48,4 +51,9 @@ public class CustomsReference {
   @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
   private ConsignmentItem consignmentItem;
 
+  @ToString.Exclude
+  @EqualsAndHashCode.Exclude
+  @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+  private CargoItem cargoItem;
+
```

Comment:
As far as I can tell, this attribute is not being used. It is explicitly **not** mapped when mapping from the TO to the DAO.  If the code works without this attribute, please remove it.

(Verify that the CustomsReference on the CargoItem remain on the `GET` call)